src/api/services/slack_notify/notification.py

```diff
@@ -43,4 +43,11 @@ def send_message(self, message, slack_id):
             user=slack_id,
             as_user=True,
             icon_emoji=':ninja:',
-        )
\ No newline at end of file
+        )
+
+    def send_notification(self, roles, users, message):
+        for role in roles:
```

Comment:
```suggestion
        This still works out fine but can it be refactored using list/dict comprehension ?
```

Reply:
`roles` and `users` are different query sets. They both return two different lists, which have to contain the required properties.